Changed Colors and Styles

hariharan-mahadevan-10884 opened the pull request:
Changed background colors, radio button styles, Table colors and Submit button style,

Commit:
made changes to 1. css/light-style.css (changed the colors, button style of
submit button)
2. css/radio-button-light-style.css (change radio button styles)
3. css/main.css (removed border of the main container)

Commit:
Small change in light-style.css

Commit:
Seperated Color and other properties accordingly

Commit:
A small change

sivaraam commented on css/main.css:
```diff
@@ -195,6 +196,22 @@ td, th {
+    font-size: 20px;
+    margin: 4px 2px;
+    -webkit-transition-duration: 0.4s; /* Safari */
+    transition-duration: 0.4s;
```
What's the use of the transition-duration ? What are you transforming ?

hariharan-mahadevan-10884 replied:
Actually I changed the color of the button on hover later removed it . For that only I have used transition-duration

sivaraam commented on css/main.css:
```diff
@@ -195,6 +196,22 @@ td, th {
+    width:250px;
+    padding: 16px 32px;
+    text-align: center;
+    text-decoration: none;
```
The above two lines don't seem to have any effect, what's their intended use ?

hariharan-mahadevan-10884 replied:
Which two lines?

sivaraam replied:
text-align and text-decoration properties

sivaraam commented on css/main.css:
```diff
@@ -195,6 +196,22 @@ td, th {
+{
+    border: 2px;
+    border-radius:4px;
+    width:250px;
```
That seems too wide. 🤔

sivaraam reviewed (changes requested):
Needs changes. See if you could convert all the `px` values into `em`s. The `em` are a better than pixels as they scale well.

sivaraam committed:
Merge branch 'validation' into radio-to-text

Commit:
1.Added drop down for subject handled.
2.Included a new image for drop down background
3.Added a row to the table which contains check boxes for selecting the
period.
4.Created 2 new css files for the check boxes.
5.Changed px values to em values
6.Changed colors and some styles

Commit:
Merge branch 'radio-to-text' of https://github.com/harideepan/Seminar into radio-to-text

Commit:
Changed px values to em values

sivaraam commented on css/light-styles.css:
```diff
@@ -37,21 +38,48 @@ select {
 
 table {
-  border-color: #E24E42;
+  border:none;
```
I remember telling you not to keep anything other than color and font-related rules in this file.

sivaraam commented on css/light-styles.css:
```diff
@@ -37,21 +38,48 @@ select {
 table th, table tr {
-  border-color: #fff;
-  background-color: #0A1612;
+  border:none;	
```
Same here.

sivaraam commented on css/light-styles.css:
```diff
@@ -37,21 +38,48 @@ select {
+  background-color: #07889B;
+}
+
+.zeroPadding
```
Give a better name than that.

sivaraam replied:
And yes move it to the main.css file

sivaraam commented on css/main.css:
```diff
@@ -143,7 +143,6 @@ html, body {
   text-align: left;
   padding-top: 1em;
   margin-right: 1em;
-  transition: border 0.5s linear;
```
This was there for a reason. Bring it back

sivaraam commented on css/main.css:
```diff
@@ -160,30 +159,19 @@ html, body {
 /* Hide radio dept selector for small screens */
-.halls-radio-container {
+#halls-radio {
   display: none;
```
Reason for changing the above classes into IDs ?

sivaraam commented on css/main.css:
```diff
@@ -160,30 +159,19 @@ html, body {
   display: block;
 }
 
-/* Hide the status table and book button until the user selects
```
Don't remove it. Comment it for the mean time.

sivaraam commented on css/main.css:
```diff
@@ -208,22 +196,59 @@ td, th {
-    font-size: 20px;
-    margin: 4px 2px;
+    font-size: 1.35em;
+    margin:0em;
```
I guess you could remove the margin property. It defaults to 0, I guess.

sivaraam commented on index.html:
```diff
@@ -15,7 +15,9 @@
-
+		<link rel="stylesheet" href="css/check-box.css">
+		<link rel="stylesheet" href="css/check-box-light-style.css">
+		
```
Add `type` attributes to `link` elements

sivaraam commented on index.html:
```diff
@@ -41,26 +43,24 @@ <h1 class="title">Seminar Hall Booking</h1>
-                          <option value="select"> Select </option>
+                      <div id="date-select">
+                        <select id="dates" name="date">
                           <option value="20">20 Thu</option>
```
The default value should be `Select` for a reason. Further don't change the `ID`s and `name` attributes they are meant to be what they are.

sivaraam commented on index.html:
```diff
@@ -97,7 +97,7 @@ <h1 class="title">Seminar Hall Booking</h1>
                         <div class="css-button date-button-col">
-                          <input type="radio" name="date-radio" id="date-5" value="">
+                          <input type="radio" name="date" id="date-5">
                           <div class="css-button-label-container">
```
Revert the names to their old values and don't change the name and ID's in the future. _They are meant to be what they are_.

sivaraam commented on index.html:
```diff
@@ -118,15 +118,8 @@ <h1 class="title">Seminar Hall Booking</h1>
-                          <option value="select">Select</option>
+                      <div id="halls-select">
+                        <select id="halls" name="hall">
                           <option value="cse">  CSE </option>
```
Don't remove comments and do changes as stated before

sivaraam commented on index.html:
```diff
@@ -206,8 +199,27 @@ <h1 class="title">Seminar Hall Booking</h1>
                   </div> <!-- .form-item -->
                 </div>  <!-- .form-item-container -->
-
-                <div id="status-table-container" class="form-item-container">
```
Be careful while adding new items. You seemed to have removed the id of the table container. Bring it back.

sivaraam commented on index.html:
```diff
@@ -219,52 +231,148 @@ <h1 class="title">Seminar Hall Booking</h1>
-                              <th> II </th>
-                              <th> III </th>
-                              <th> IV </th>
-                            </tr>
```
The `th` elements should be a child of `tr` element

sivaraam commented on index.html:
```diff
@@ -219,52 +231,148 @@ <h1 class="title">Seminar Hall Booking</h1>
+								</div>
+								</div>
+ 							</td>
+							</tr>
```
Fix the white space issues. They look too odd.

sivaraam commented on index.html:
```diff
@@ -219,52 +231,148 @@ <h1 class="title">Seminar Hall Booking</h1>
+								</div>
+								</div>
+ 							</td>
+							</tr>
```
White space issues again

sivaraam commented on index.html:
```diff
@@ -219,52 +231,148 @@ <h1 class="title">Seminar Hall Booking</h1>
 
-                <div id="button-container" class="form-item-container">
+                <div class="form-item-container">
                   <div class="form-item button">
```
Don't touch the ID's

sivaraam reviewed (changes requested):
There seems to be lots of changes than the title of the Pull request states. Keep changes focused to the title of Pull request. Create new PRs for separate changes. 

You seem to using tabs for indentation. Switch to spaces. See how to make your text editor to insert spaces when you hit tab. The behaviour should be 4spaces for a single tab.

sivaraam commented:
@sirganeshv There seems to be changes in JS files. You better have a look.

sivaraam commented on css/main.css:
```diff
@@ -207,6 +196,59 @@ td, th {
+{
+    border: 0.125em solid;
+    border-radius:0.25em;
+    width:9.375em;
```
As stated in previous review. Doesn't that seem too wide.

sivaraam commented:
A few things I missed,

1. Change the title and description of this PR to reflect what it changes. In future, create separate PRs for separate changes.
2. Create more commits. Don't try to put every change you do into a single commit. It clutters the purpose of version control
3. The subject selection doesn't look good.
![screenshot from 2017-04-23 08-10-38](https://cloud.githubusercontent.com/assets/12448084/25310555/40664050-27d6-11e7-95bd-1b4cece68e39.png)

sivaraam commented:
One more important thing, I remember telling you to give the theme that  **resembles** the one found at the [Sass homepage](http://sass-lang.com/). Change the colors so that they **resemble** the homepage.
